ecl/hqlcpp/hqlsource.cpp

```diff
@@ -2009,6 +2009,18 @@ ABoundActivity * SourceBuilder::buildActivity(BuildCtx & ctx, IHqlExpression * e
         instance->addAttribute("spillReason", text.str());
     }
 
+    if (tableExpr)
+    {
+        IHqlExpression * signedAttr = tableExpr->queryAttribute(_fileAccessSigned_Atom);
+        if (signedAttr)
+        {
+            IValue *sig = signedAttr->queryChild(0)->queryValue();
+            StringBuffer buf;
+            sig->getStringValue(buf);
```

Comment:
getStringValue(buf, signedAttr->queryChild(0)) is simpler.
Also, does this need to be utf-8?  Probably not.